apis/core/v1beta1/conditions.go

```diff
@@ -321,4 +324,13 @@ const (
 
 	// OpenStackControlPlaneCeilometerReadyErrorMessage
 	OpenStackControlPlaneCeilometerReadyErrorMessage = "OpenStackControlPlane Ceilometer error occured %s"
+
+	// OpenStackControlPlaneServiceOverrideReadyInitMessage
+	OpenStackControlPlaneServiceOverrideReadyInitMessage = "OpenStackControlPlane service override not started"
+
+	// OpenStackControlPlaneServiceOverrideReadyMessage
+	OpenStackControlPlaneServiceOverrideReadyMessage = "OpenStackControlPlane service overrides completed"
+
+	// OpenStackControlPlaneServiceOverrideReadyErrorMessage
+	OpenStackControlPlaneServiceOverrideReadyErrorMessage = "OpenStackControlPlane %s service endpoint %s override error occured %s"
```

Comment:
nit: I think we should try to use the same pattern here for the name of item, i.e. instead of "service override" vs "service overrides" vs "service endpoint override".  Doesn't matter which one I think.

Reply:
I mean just the assigned text.  Each one has a different name:

- `service override`
- `service overrides`
- `service endpoint %s override`

Maybe the last one needs to be unique given that you're injecting something into it.  Not a big deal anyhow.

Reply:
done